nixos/modules/services/web-servers/nginx/vhost-options.nix

```diff
@@ -4,10 +4,28 @@
 # the user/group to run under.)
 
 { config, lib, ... }:
+let
+  global-config = config;
+in
+{ config, name, ... }:
```

Comment:
@roberth @infinisil I'm curious, are there any best practices on importing submodules from other files? This seems kinda odd, but right now I don' thave a better idea.

Reply:
To clarify, importing into submodules is ok and lets you do interesting things, and it may even look like this, but this isn't the right use case for it, because option values shouldn't be forwarded en masse to become definitions for multiple options.

Reply:
> shouldn't be forwarded en masse to become definitions for multiple options.

Agreed, that's why I opened #313412 and I even have a half-finished thing in my stash.

That said, this is out-of-scope here, it contains a nice improvement and as long as we don't break existing code here, we're good I'd say.